libraries/postgres/src/Schema.js

```diff
@@ -39,14 +37,14 @@ class Schema{
 
       const version = yaml.safeLoad(fs.readFileSync(filename));
 
-      versions.set(version.version, version);
+      versions[version.version - 1] = version;
```

Comment:
We could avoid the `-1` by putting a "dummy" version 0 in the array.  I don't know if that would make things clearer or not :)